src/dropdown/Dropdown.js

```diff
@@ -90,12 +93,13 @@ export default class Dropdown extends React.Component {
   getOnChange = () => this.props.valueLink ? this.props.valueLink.requestChange : this.props.onChange;
 
   getCustomClassNames() {
-    const { size, flat, clearable } = this.props;
+    const { size, flat, clearable, menuPosition } = this.props;
     return cx({
       'is-medium': size === 'medium',
       'is-small': size === 'small',
       'is-flat': flat,
-      'is-clearable': clearable
+      'is-clearable': clearable,
+      top: menuPosition === 'top'
```

Comment:
since the `className` is attached to the top-level component, would it be better to use a more self explaining name? like `has-menu-position-on-top`?